Project.toml

```diff
@@ -3,11 +3,18 @@ uuid = "b3c78db8-5351-4c94-92cb-e6a7a02f3abe"
 authors = ["JieLi"]
 version = "0.1.0"
 
+[deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+NiLang = "ab4ef3a6-0b42-11ea-31f6-e34652774712"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+
 [compat]
 julia = "1"
```

Comment:
Generally, we would want every non-standard package to have associated version compat. But since we're going to rebase on top of NiLangCore, which can be left in the scope of the next PR. I wouldn't bother much here.